x-pack/plugins/enterprise_search/public/applications/workplace_search/components/overview/overview.test.tsx

```diff
@@ -18,58 +18,53 @@ import { ViewContentHeader } from '../shared/view_content_header';
 import { OnboardingSteps } from './onboarding_steps';
 import { OrganizationStats } from './organization_stats';
 import { RecentActivity } from './recent_activity';
-import { Overview, defaultServerData } from './overview';
+import { Overview } from './overview';
 
 describe('Overview', () => {
-  const mockHttp = mockKibanaContext.http;
-
   describe('non-happy-path states', () => {
     it('isLoading', () => {
       const wrapper = shallow(<Overview />);
 
       expect(wrapper.find(Loading)).toHaveLength(1);
     });
 
-    it('hasErrorConnecting', async () => {
-      const wrapper = await mountWithAsyncContext(<Overview />, {
-        http: {
-          ...mockHttp,
-          get: () => Promise.reject({ invalidPayload: true }),
-        },
-      });
+    it('hasErrorConnecting', () => {
+      setMockValues({ hasErrorConnecting: true });
+      const wrapper = shallow(<Overview />);
 
       expect(wrapper.find(ErrorState)).toHaveLength(1);
     });
   });
 
   describe('happy-path states', () => {
-    it('renders onboarding state', async () => {
-      const mockApi = jest.fn(() => defaultServerData);
-      const wrapper = await mountWithAsyncContext(<Overview />, {
-        http: { ...mockHttp, get: mockApi },
-      });
+    it('calls initialize function', async () => {
+      mount(<Overview />);
+
+      expect(mockLogicActions.initializeOverview).toHaveBeenCalled();
```

Comment:
Really interesting how much simpler this test suite got with the Kea changes! I'm kind of hand-wavy right now as to whether that's a good or bad thing, but definitely cool to see the code diff at least - I wonder if we'll even need `mountWithAsyncContext` after Kea